modules/tinymce/src/themes/silver/main/ts/ui/context/ContextToolbar.ts

```diff
@@ -173,15 +170,14 @@ const register = (editor: Editor, registryContextToolbars: Record<string, Contex
   const getAnchor = (position: InlineContent.ContextPosition, element: Optional<SugarElement<Element>>): AnchorSpec => {
     const anchorage = position === 'node' ? sharedBackstage.anchors.node(element) : sharedBackstage.anchors.cursor();
     const anchorLayout = ContextToolbarAnchor.getAnchorLayout(editor, position, isTouch(), {
-      lastPos: lastPosition.get,
       lastElement: lastElement.get,
       isReposition: () => Optionals.is(lastTrigger.get(), TriggerCause.Reposition),
       bounds: lastBounds.get
     });
     return Merger.deepMerge(anchorage, anchorLayout);
   };
 
-  const launchContext = (toolbarApi: Array<ContextType>, elem: Optional<Element>) => {
+  const launchContext = (toolbarApi: Array<ContextType>, elem: Optional<SugarElement<Element>>) => {
```

Comment:
We were unwrapping this and immediately rewrapping this as a `SugarElement` so this just removes that unnecessary action.